api/docker-compose.yaml

```diff
@@ -133,6 +151,24 @@ services:
       - GOOGLE_APPLICATION_CREDENTIALS=/w/sa-key.json
     env_file:
       - db-cdr/vars.env
+  db-generate-cloudsql-db:
```

Comment:
Note for follow-up - I don't think you need 1 entry here per shell script , the configuration across all of these CDR commands are identical save for the script. I think you could probably get away with just one service here like "cdr-scripts", then invoke it like:

```
docker-compse run cdr-script my-script.sh
```

at the callsites. Possibly one additional target needed for those that access a local DB, e.g. cdr-local-scripts. I'd keep this as-is for this PR for now though.